nemo/collections/nlp/modules/common/__init__.py

```diff
@@ -21,7 +21,11 @@
     DistilBertEncoder,
     RobertaEncoder,
 )
-from nemo.collections.nlp.modules.common.lm_utils import get_lm_model, get_pretrained_lm_models_list
+from nemo.collections.nlp.modules.common.lm_utils import (
+    get_lm_model,
+    get_megatron_lm_models_list,
+    get_pretrained_lm_models_list,
```

Comment:
Is get_pretrained_lm_models_list for HF and get_megatron_lm_models_list for Megatron? If yes, how about naming get_pretrained_lm_models_list to get_hf_lm_models_list?

Reply:
get_pretrained_lm_models_list this includes both HF and megatron